ad-tag/source/ts/ads/criteo.ts

```diff
@@ -0,0 +1,89 @@
+import type { MoliRuntime } from 'ad-tag/types/moliRuntime';
+import { prebidjs } from 'ad-tag/types/prebidjs';
+import { headerbidding } from 'ad-tag/types/moliConfig';
+import IUserSyncConfig = prebidjs.userSync.IUserSyncConfig;
+
```

Comment:
This trips the schema.json generation some times. Use fully qualified import paths instead.